NBitcoin/Protocol/Behaviors/BroadcastTransactionBehavior.cs

```diff
@@ -254,7 +253,7 @@ private void Announce(TransactionBroadcast tx, uint256 hash)
 			{
 				tx.State = BroadcastState.Announced;
 				tx.AnnouncedTime = DateTime.UtcNow;
-				var unused = node.SendMessageAsync(new InvPayload(InventoryType.MSG_TX, hash)).ConfigureAwait(false);
+				node.SendMessageAsync(new InvPayload(InventoryType.MSG_TX, hash)).ConfigureAwait(false);

```

Comment:
on the contrary this line if used give me warning whereas with unused I don't have any